### DISCUSSION TITLE
how can i use closeToast inside the toast itself.

### DISCUSSION THREAD

**Discussion by LakshayjeetSingh** [fkhadra/react-toastify#933] [Q&A]
I want to dismiss the toast from with-in like closeButton but with my own button but i don't see any API for that. I cannot use toast.dismiss() as it dismisses all the toast

**Comment by ChrisBoon** [comment]
The docs show an answer for this as of v10:
https://fkhadra.github.io/react-toastify/render-what-you-want
"When you render a component, a closeToast prop and the toastProps are injected into your component."
